packages/analyzer/src/features/analyse-phase/creators/createVariable.js

```diff
@@ -12,7 +14,11 @@ export function createVariable(variableStatementNode, declarationNode) {
     variableTemplate.type = { text: declarationNode?.type?.getText() }
   }
 
+  if (declarationNode.initializer && (isAsConst(declarationNode.initializer) || ts.isPropertyAccessExpression(declarationNode.initializer))) {
```

Comment:
maybe it makes sense to abstract this into a separate fn as well?

Reply:
yeah, couldn't think of a good name though. thoughts?

Reply:
Hrm. Same. Not great to have `isAsConst` and `hasAsConst` or `containsAsConst` 🙈  Maybe make the current `isAsConst` an implementation detail of this function?

Reply:
ok